### PR TITLE
reference/performance: update docs for plan bind

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? 

Update docs for plan bindings.

### What is the related PR or file link(s)? 

- Reference link(s):
- This PR is to align with: https://github.com/pingcap/docs-cn/pull/2250
- [ ] N/A (not applicable)

**Important Notice:**

**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:

- **needs-cherry-pick-3.0**
- **needs-cherry-pick-3.1**
- **needs-cherry-pick-2.1**

### CODE REVIEW

File: reference/performance/execution-plan-bind.md [hunk 1]
```diff
@@ -76,3 +76,29 @@ This statement outputs the execution plan bindings at the GLOBAL or SESSION leve
...
+
+> **Note:**
+>
+> The auto binding creation function relies on [Statement Summary](/reference/performance/statement-summary.md). Hence, turn on Statement Summary before using auto binding.
+
+After enabling auto binding creation, for every `bind-info-lease` (3 seconds by default), the historical SQL statements in Statement Summary will be traversed, and the SQL statements that appear at least twice is automatically bound.
+
+### Automatically evolve binding
+
+As the data updates, the previously bound execution plan may no longer be optimal. The automatic binding evolution function can optimize the bound execution plan.
```
Comment (lilin90) [inline reference/performance/execution-plan-bind.md]:
```suggestion
As data updates, the previously bound execution plan might no longer be optimal. The automatic binding evolution feature can optimize the bound execution plan.
```

File: reference/performance/execution-plan-bind.md [hunk 2]
```diff
@@ -76,3 +76,29 @@ This statement outputs the execution plan bindings at the GLOBAL or SESSION leve
...
+
+{{< copyable "sql" >}}
+
+```sql
+set global tidb_evolve_plan_baselines = on;
+```
+
+The default value of `tidb_evolve_plan_baselines` is `off`.
+
+After enabling the automatic binding evolution function, if the optimal execution plan selected by the optimizer is not among the bound execution plans, the optimizer marks the plan as an execution plan that waits for verification. For every `bind-info-lease` (3 seconds by default), an execution plan to be verified is selected and compared with a bound execution plan with the least cost, in terms of the actual execution time. If the plan to be verified is better, it is marked as a usable binding.
```
Comment (lilin90) [inline reference/performance/execution-plan-bind.md]:
```suggestion
After the automatic binding evolution feature is enabled, if the optimal execution plan selected by the optimizer is not among the bound execution plans, the optimizer marks the plan as an execution plan that waits for verification. Every `bind-info-lease` (the default value is `3s`), an execution plan to be verified is selected and compared with a bound execution plan with the least cost, in terms of the actual execution time. If the plan to be verified is better, it is marked as a usable binding.
```

File: reference/performance/execution-plan-bind.md [hunk 3]
```diff
@@ -76,3 +76,29 @@ This statement outputs the execution plan bindings at the GLOBAL or SESSION leve
...
+
+To reduce the impact that the automatic evolution has on clusters, you may limit the max time of each execution plan by setting `tidb_evolve_plan_task_max_time`. The default value is `600s`. You may limit the time window by setting `tidb_evolve_plan_task_start_time` (`00:00 +0000` by default) and `tidb_evolve_plan_task_end_time` (`23:59 +0000` by default).
```
Comment (lilin90) [inline reference/performance/execution-plan-bind.md]:
```suggestion
To reduce the impact that the automatic evolution has on clusters, use the following configurations:

- Set `tidb_evolve_plan_task_max_time` to limit the maximum execution time of each execution plan. The default value is `600s`.
- Set `tidb_evolve_plan_task_start_time` (`00:00 +0000` by default) and `tidb_evolve_plan_task_end_time` (`23:59 +0000` by default) to limit the time window.
```

**Comment by ran-huang** [comment]
@lilin90 PTAL~

**Pull request merged**